### PR TITLE
Create handle add blog (one)

### PR DESCRIPTION
**Pull request by DuongPham-dev**
- #11 
- Since there are too many files change, I will split it into 2 commits

### CODE REVIEW

File: practices/practice-one/src/components/Header/Header.tsx [hunk 1]
```diff
@@ -7,18 +7,52 @@ import Paragraph from "components/Paragraph/Paragraph";
...
   const [popup, setPopup] = useState(false);
 
-  const handleSubmit = (event: FormEvent) => {};
+  const { blog } = state;
+
+  const handleSubmit = (event: FormEvent) => {
+    event.preventDefault();
+    const { dataFields, error } = validation(blog);
+    if (error) {
+      // I will add the error message later
```
Comment (cunguyendev) [inline practices/practice-one/src/components/Header/Header.tsx]: // TODO: I will add the error message later

File: practices/practice-one/src/components/Header/Header.tsx [hunk 2]
```diff
@@ -7,18 +7,52 @@ import Paragraph from "components/Paragraph/Paragraph";
...
   };
 
+  const handleSetValueBlog = (event: ChangeEvent) => {
+    type InputType = HTMLInputElement | HTMLSelectElement;
+    const element: InputType = event.target as InputType;
+    const key: string = element.name;
+    let value: string = element.value;
+    if (element.type === "file") {
+      const fileElement: HTMLInputElement = element as HTMLInputElement;
+      const file = fileElement.files ? fileElement.files[0] : new File([], "temp.jpg");
```
Comment (cunguyendev) [inline practices/practice-one/src/components/Header/Header.tsx]: Why temp.jpg ?

File: practices/practice-one/src/helpers/validation/empty.ts [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+ * - Used to check the values ​​according to the condition.
+ * @param object Contains keys as fields to be checked, values ​​as data to be checked
+ * @param regex Contains regular expressions, with the key being the corresponding data field to be checked
+   the value will contain the regular expression and the message when there is an error
+ * @returns 
+ */
+const validation = (object: ValidationProps, regex?: RegexProps): ValidateReturn => {
+    let dataFields = {}
+    let error = false
+    Object.entries(object).forEach(([key, value]) => {
```
Comment (cunguyendev) [inline practices/practice-one/src/helpers/validation/empty.ts:28]: Can we refactor this by map ?

File: practices/practice-one/src/components/Header/Header.tsx (hunk 1 above)
Comment (DuongPham-dev) [reply practices/practice-one/src/components/Header/Header.tsx]: I will update

File: practices/practice-one/src/helpers/validation/empty.ts (hunk 3 above)
Comment (DuongPham-dev) [reply practices/practice-one/src/helpers/validation/empty.ts:28]: I just want to get the invalid fields, for the sake of showing errors, so I think I won't use map in this case.

File: practices/practice-one/src/components/Header/Header.tsx (hunk 2 above)
Comment (DuongPham-dev) [reply practices/practice-one/src/components/Header/Header.tsx]: I don't understand File() well, so I put it by myself, can you suggest me

**Pull request closed without merging**